docs/rst/manual/theory/ensemble_based_methods.rst

```diff
@@ -397,9 +397,97 @@ Iterative Ensemble Smoother - Ensemble subspace version
 
 
+Kalman Posterior Properties
+---------------------------
 
+The updating from the prior :math:`p(\psi)=N\left(\mu_\psi,\Sigma_\psi\right)` 
+to the posterior :math:`p(\psi|d)=N\left(\mu_{\psi|d},\Sigma_{\psi|d}\right)`, assimilating measurements :math:`d`, 
+linear in :math:`\psi` is performed by the Kalman methods by employing the following equations
 
+.. math::
+   \begin{align}
+   \mu_{\psi|d} &= \mu_{\psi} + K(d-M\mu_{\psi}),\\
+   \Sigma_{\psi|d} &= (I-KM) \Sigma_{\psi}
+   \end{align}
+
+where
+
+.. math::
+   \begin{align}
+   K = \Sigma_{\psi}M^\top (M\Sigma_{\psi}M^\top + \Sigma_{d})^{-1}
+   \end{align}
+
+is called tha Kalman gain, and :math:`M` is the linear measurement matrix, so that
+
+.. math::
+   \begin{align}
+   \hat{d} = M\mu_{\psi}
+   \end{align}
+
+is the best estimate of :math:`d` under the prior knowledge, and the error is assumed Gaussian with covariance :math:`\Sigma_d`. 
+The ensemble variants draw an :math:`N`-sample :math:`\{\psi\}_{i=1}^N` from the prior, 
+and perturbs observations :math:`d` using the distributions of measurements creating a corresponding observation-sample :math:`\{d\}_{i=1}^N`.
+The perturbations are guaranteed to sum to zero over the sample.
+A posterior sample is then formed from updating the prior with the equation for the posterior mean above
+
+.. math::
+   \begin{align}
+   \{\psi_i | d_i\} = \psi_i + \hat{K}(d_i-M \psi_i),
+   \end{align}
+
+where the estimated Kalman gain :math:`\hat{K}` is found by exchanging the prior covariance with an estimate based on its sample.
+Thus, the ensemble methods combine a sample from the prior with a sample from the likelihood of observed data, to form a new sample from the posterior.
+The posterior distribution that the posterior sample is conseptually sampled from, has mean and covariance found by
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{\psi|d} &= \bar{\psi} + \hat{K}(\bar{d}-M\bar{\psi}),\\
+   \hat{\Sigma}_{\psi|d} &= (I-\hat{K}M) \hat{\Sigma}_{\psi}
+   \end{align}
+
+From this, it is seen that when the sample size tends to infinity and estimates converge to the corresponding population quantities,
+then the ensemble variants converge to the standard Kalman filter in the linear Gaussian case.
+This convergence is however of a stochastic nature.
+
+More deterministic properties of the posterior are observed when the belief in measurements :math:`d` is varied. 
+Intuitively, when measurements have zero belief, i.e. unbounded variance, then the posterior should equal the prior.
+At the other end of the spectrum, if the measurements are perfect with zero variance,
+then the posterior estimate should equal the maximum-likelihood estimate, corresponding to a flat prior, 
+and as we are certain of the belief in this estimate (because the measurements are so amazing), 
+the determinant of the posterior covariance tends to zero from above.
```

Comment:
To me this implies that the determinant of a covariance matrix can be negative, which I don't think is the case.

Reply:
The eigenvalues are all real and positive for a covariance matrix (due to symmetry, I think), so your are correct.
Is it the mentioning of "from above" that gives the implication?
I don't see the implication, 🤔 can you elaborate?

Reply:
Yeah I was parsing it wrong, it looks good!